widgetssdk/src/main/java/com/glia/widgets/UiTheme.java

```diff
@@ -711,8 +620,19 @@ public void setIconUpgradeAudioDialog(@DrawableRes Integer iconUpgradeAudioDialo
             this.iconUpgradeAudioDialog = iconUpgradeAudioDialog;
         }
 
+        @Deprecated
```

Comment:
Sorry if it is obvious, but I don't understand why only some methods related to styling are deprecated. For example, why `setIconCallChat` method is deprecated and `setIconChatAudioUpgrade` is not. Or is it just that they are not related to the call screen?